components/brave_perf_predictor/browser/bandwidth_savings_predictor_unittest.cc

```diff
@@ -0,0 +1,154 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_perf_predictor/browser/bandwidth_savings_predictor.h"
+
+#include "base/containers/flat_map.h"
+#include "base/time/time.h"
+#include "chrome/browser/predictors/loading_test_util.h"
+#include "components/page_load_metrics/common/page_load_metrics.mojom.h"
+#include "components/page_load_metrics/common/page_load_timing.h"
+#include "content/public/common/resource_load_info.mojom.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "url/gurl.h"
+
+namespace brave_perf_predictor {
+
+TEST(BandwidthSavingsPredictorTest, FeaturiseBlocked) {
+  auto* predictor = new BandwidthSavingsPredictor();
```

Comment:
oops, this is a memory leak. I guess you can just make it a stack variable?
`BandwidthSavingsPredictor predictor;`